### PR TITLE
[Messenger] make StopWorkerOnSignalsListener listen by default on SIGTERM and SIGINT

### LINKED ISSUE #49156: [Messenger] Also listen to `SIGINT` in `StopWorkerOnSigtermSignalListener`
### Description

ATM, [StopWorkerOnSigtermSignalListener](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Messenger/EventListener/StopWorkerOnSigtermSignalListener.php#L32) listen only `SIGTERM`. 

IMHO, it should also listen SIGINT (CTR+C) for the following reasons:

* in dev, we can have the same behavior than in production
* in prod, when adding manually more consumers (yeah, SSH + `bin/console` 🧓🏼), it's boring to stop them cleanly. CTRL+C is easier
* I always found it weird it doesn't listen SIGINT - Okay, this is not a valid argument :) 

cc @Tobion 

---

I'll make the PR if accepted

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | yes
| Tickets       | Fix #49156
| License       | MIT
| Doc PR        | no need I guess

### CODE REVIEW

**Comment by lyrixx** [comment]
This PR is ready for review 👍🏼

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php [hunk 1]
```diff
@@ -218,4 +214,24 @@
...
+        $services
+            ->set('messenger.listener.stop_worker_signals_listener', StopWorkerOnSignalsListener::class)
+                ->args([
+                    null,
+                    service('logger')->ignoreOnInvalid(),
+                ])
+                ->tag('kernel.event_subscriber')
+        ;
+    } else {
+        // BC layer for symfony/messenger < 5.3
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php]: you mean 6.3 I suppose
but let's bump the min version of messenger instead?

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php]: looks like it's already at 6.3 so this can be removed

Comment (lyrixx) [reply src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php]: Oups 😅 + Good catch. Fixed 👍🏼

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**